charts/illumidesk/values.yaml

```diff
@@ -367,32 +367,11 @@ postgresql:
 datadog:
   # Enables or disables the Datadog agent
   enabled: false
-  # Datadog's configuration settings
-  datadog:
-    # API key for the DatadogHQ account
-    apiKey: ''
-    # Customer's site
-    site: ''
-    # The Kubernetes cluster name
-    clusterName: ''
-    # Logs configuration
-    logs:
-      # Enables or disables log collection
-      enabled: true
-      # If true, collects all logs from running containers.
-      containerCollectAll: true
-    # Datadog's cluster agent which acts as a proxy between Datadog's API and
-    # node-based agents.
-    # ref: https://docs.datadoghq.com/agent/cluster_agent/
-    clusterAgent:
-      # Enables or disables the cluster agent
-      enabled: true
-      token: ''
-      # Metrics enables autoscaling workloads based off of Datadog metrics
-      # ref: https://docs.datadoghq.com/agent/cluster_agent/external_metrics/
-      metricsProvider:
-        # Enables or disables external metrics with Datadog.
-        enabled: true
+  # API Key created for your Datadog instance
+  # ref: https://docs.datadoghq.com/account_management/api-app-keys/
+  apiKey:
+  # Create Random Token using base64encoding and pass it here for the cluster agent
+  clusterAgentToken:
```

Comment:
I see references to .Values.datadog.namespace in the manifests but don't see it in values.yaml.